src/dashboard/src/installer/steps.py

```diff
@@ -37,10 +37,14 @@
 
 def create_super_user(username, email, password, key):
     UserModel = get_user_model()
-    user = UserModel._default_manager.db_manager('default').create_superuser(username, email, password)
-    api_key = ApiKey.objects.create(user=user)
-    api_key.key = key
-    api_key.save()
+    # Create the new super user if it doesn't already exist
+    try:
+        user = UserModel._default_manager.get(**{UserModel.USERNAME_FIELD: username})
```

Comment:
I've found there is a method in the base class for this particular use case: 

    user = self.UserModel._default_manager.db_manager(database).get_by_natural_key(username)

Feel free to ignore this. I almost prefer to see the kwargs in place.

Reply:
That bit of code is exactly as it is in [create_user.py](https://github.com/JiscRDSS/archivematica-storage-service/blob/qa/jisc/storage_service/common/management/commands/create_user.py) in the Storage Service, I just reused it, since the SS was working as required and the Dashboard wasn't.